packages/compiler/test/formatter/formatter.test.ts

```diff
@@ -680,7 +680,7 @@ enum Bar {
       });
     });
 
-    it("keeps comment between statements of a", () => {
+    it("keeps comment between statements of a flatten namespace", () => {
       assertFormat({
         code: `
         namespace Foo.Bar {
```

Comment:
Is there test coverage already for commented flattened namespace with decorators?

Reply:
yeah seems to have issue there too, will check